modular_skyrat/modules/goofsec/code/department_guards.dm

```diff
@@ -180,8 +180,8 @@
 		/obj/item/food/donut/caramel = 10,
 		/obj/item/food/donut/matcha = 10,
 		/obj/item/food/donut/blumpkin = 5,
-		/obj/item/clothing/mask/whistle = 5,
-		/obj/item/melee/baton/security/boomerang/loaded = 1
+		/obj/item/clothing/mask/whistle = 10,
+		/obj/item/melee/baton = 5
```

Comment:
ditto, like all the other dep guards

Reply:
Ah I forgot that one. Thanks for pointing it out